profile/README.md

```diff
@@ -0,0 +1,31 @@
+## Hi there 👋
+
+🙋‍♀️ OpenSCD product vision
+
+For utility/integrator/vendor companies​
+
+Who want to have flexibility, agility and efficiency in their SCL configurations​
+
+**OpenSCD**​
+
+is an IEC 61850 SCL configuration tool​
+
+That offers a front-end, open source, extensible, customizable, user friendly, SCL editing​
+
+Unlike proprietary, vendor specific, desktop oriented, limited integration options, not user friendly tools ​
```

Comment:
I do not have a good feeling with this sentence. It does sound a bit as if this is the case with all other tools. I am not sure if this negation is really required to describe what OpenSCD is. I would rather concentrate on what it is and not what it not is.

Reply:
I tried to use a product vision template here: https://www.prodpad.com/resources/free-product-vision-template/ ; I think is makes sense to describe what sets OpenSCD apart.